### ISSUE TITLE
[Bug] Really slow media downloads (AnkiDroid 2.15alpha48)

### ISSUE THREAD

**Issue by phly95** [ankidroid/Anki-Android#8765]
###### Reproduction Steps

1. On desktop, download a large deck like https://ankiweb.net/shared/info/957579516
2. sync it to ankiweb
3. attempt to sync on Ankidroid

###### Expected Result

downloads all media files in a couple of minutes

###### Actual Result

Downloads at a molasses pace of about 25 media files every 30 seconds or so.

###### Debug info
Refer to the [support page](https://ankidroid.org/docs/help.html) if you are unsure where to get the "debug info".

###### Research
*Enter an [x] character to confirm the points below:*

- [x] I have read the [support page](https://ankidroid.org/docs/help.html) and am reporting a bug or enhancement request specific to AnkiDroid
- [x] I have checked the [manual](https://ankidroid.org/docs/manual.html) and the [FAQ](https://github.com/ankidroid/Anki-Android/wiki/FAQ) and could not find a solution to my issue
- [x] I have searched for similar existing issues here and on the user forum
- [x] (Optional) I have confirmed the issue is not resolved in the latest alpha release ([instructions](https://docs.ankidroid.org/manual.html#betaTesting))

**Comment by david-allison-1** [comment]
Is this a regression from AnkiDroid 2.14.6? We've fallen behind in upgrades on the media syncing, and we'll be working on this as part of the Rust migration

**Comment by phly95** [comment]
I'm not sure because I've upgraded my desktop Anki to the latest settings that are not compatible with 2.14.6.

**Comment by phly95** [comment]
I just changed the timezone handling to test 2.14.6, and I'm having slow downloads there too.

**Comment by phly95** [comment]
https://www.youtube.com/watch?v=TpGX20pt4Bc Here's a video of what's happening

**Comment by mikehardy** [comment]
It syncs fast if you have a fast connection and if ankiweb is behaving well, I've tested this quite recently when  worked on #8363 (and related #7465) additionally last time I was in there I recall verifying we were re-using the encrypted connection so it shouldn't be possible to optimize this much further. It is *possible* we can grab more than 25 files at a time, I'm not sure how many Anki Desktop goes for, but that's the last remaining optimization I think

In other words, I don't *think* there's any more speedup we may do internally, I believe this is an external network or server issue

**Comment by phly95** [comment]
Anyways, I went back to stock, and syncing is fast again. So the problem was on my end, not with the app. Guess I'm stuck on the February 2020 build of Android 10 for this abandoned phone.
Thank you for the amazing app! It's been incredibly useful for my studies.

**Comment by david-allison-1** [comment]
@phly95 Thanks! I'm going to close this as I don't think there's anything that we can explicitly do if it's your ROM at fault.

Working on media syncing is on the priority list, likely later this year

Best,

David
